### PR TITLE
Enhancement: Provide CheckCommand definition for Icinga 2 ITL

### LINKED ISSUE #110: Provide CheckCommand definition for Icinga 2 ITL
## Expected Behavior
The module should provide a CheckCommand definition for the integrated checks for the Icinga 2 template library.

## Your Environment

* VMware vCenter®/ESXi™-Version: independent
* Version/GIT-Hash of this module: v1.0.3
* Icinga Web 2 version: 2.6.3
* Operating System and version: CentOS 7.6
* Webserver, PHP versions: Apache, PHP 7.1.8

### PR DESCRIPTION
**Pull request by mwaldmueller**
fixes #110

### CODE REVIEW

File: contrib/itl/icingacli.conf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                "--check" = {
+                        value = "$icingacli_vspheredb_name$"
+                }
+        }
+}
+
+object CheckCommand "icingacli-vspheredb-datastore" {
+        import "icingacli-vspheredb"
+
+        command += [ "director", "check", "datastore" ]
```
Comment (dnsmichi) [inline contrib/itl/icingacli.conf]: Is `director` intended here?

**Comment by dnsmichi** [comment]
Imho this should be moved into icinga/icinga2 side by side with the other icingacli commands.

**Comment by mwaldmueller** [comment]
@dnsmichi Imho this should be in the ITL and in the module. In the module for older Icinga versions that don't provide it by ITL.

**Comment by dnsmichi** [comment]
If you're fast enough this week, this can be added into 2.11 - should be bulletproof though.

**Comment by Thomas-Gelf** [comment]
Honestly, I'm not a big fan of this - please contact me directly in case you want to know why.
@mwaldmueller: I shared my thoughts with you before you created this pull request ;-)

**Pull request closed without merging**